core/drivers/gic.c

```diff
@@ -267,6 +304,34 @@ static void gic_it_disable(struct gic_data *gd, size_t it)
 	write32(mask, gd->gicd_base + GICD_ICENABLER(idx));
 }
 
+static void gic_it_pending(struct gic_data *gd, size_t it)
```

Comment:
minor: I think `gic_it_pending()` sounds like "is IT pending?".
To ease maintenance, could it be labeled `set_it_pending()` or something explicit ?